content/docs/user-guide/networking/settings.md

```diff
@@ -5,9 +5,40 @@ linktitle: Network Settings
 ---
 
 ## Overview
-This page documents [console variables](/docs/user-guide/appendix/cvars) and settings for AzNetworking and Multiplayer behavior.
+This page documents [console variables](/docs/user-guide/appendix/cvars/) and other settings that can control Networking and Multiplayer behavior.
+
+## Networking commands
+The following [console functor](/docs/user-guide/programming/az-console/#console-functors-cfuncs) commands control the flow of networked and multiplayer games, or will be required to for a networked game or a simulation.
```

Comment:
I'm not sure what information that the phrase "or will be required to for a networked game or a simulation" is adding to the first part of the sentence. Can you explain?

Reply:
Also, "required to for" is an awkward phrase.

Reply:
Should be something "or will be required to complete setup'. Trying to highlight that LoadLevel is required here for setup but is not specific for Networked game or simulation. Will try and re-phrase, but happy to take a suggestion.

Reply:
I simplified this text.